liquidprompt

```diff
@@ -1695,6 +1698,38 @@ else
     _lp_time() { : ; }
 fi
 
+##################################
+# Detect if we're in a container #
+##################################
+
+_lp_container()
+{
+    (( LP_ENABLE_CONTAINER )) ||  return
+
+    _lp_container=""   
+
+    if [[ "$OSTYPE" == "linux-gnu" ]] ; then 
```

Comment:
`OSTYPE` is never set. It wasn't set in my test container either. I'm not sure what you are trying to compare against.

Reply:
I had seen $OSTYPE in:
* native 
     * macOS, 
     * Ubuntu, 
     * Fedora
* singularity
     * Ubuntu, 
     * Fedora
* docker
     * Ubuntu, 
     * Fedora

However, I found that $OSTYPE does not seem to be  set in Alpine-based containers.  What is your native OS?

Reply:
> I found that $OSTYPE does not seem to be set in Alpine-based containers.

Yup, that's what I tested on:
```
$ docker run -it --rm alpine:3.9 ash
/ # printf '%s\n' "$OSTYPE"

```

Why not test against `$LP_OS`?

Reply:
It's been a while since I messed with this so I think I was just using that to avoid some spurious errors on native macOS.  I don't know that I actually need the OS.